crates/uv-build/python/uv_build/__main__.py

```diff
@@ -0,0 +1,17 @@
+def main():
+    import sys
+
+    # This works both as redirect to use the proper uv package and as smoke test.
+    print(
+        "uv-build contains only the PEP 517 build backend for uv and can't be used on the CLI. "
```

Comment:
Did you want to use `uv_build` consistently?

Reply:
yes good catch